spec/i18n_js_spec.rb

```diff
@@ -422,4 +422,80 @@
       end
     end
   end
+
+  describe "translation key sorting" do
+
+    describe ".sort_translation_keys with config" do
+
+      subject { described_class.sort_translation_keys? }
+
+      context 'when :sort_translation_keys is not set in config' do
+        before { set_config "default.yml"}
+
+        it { should eq true }
+      end
+
+      context 'when :sort_translation_keys set to true in config' do
+        before { set_config "js_sort_translation_keys_true.yml"}
+
+        it { should eq true }
+      end
+
+      context 'when :sort_translation_keys set to false in config' do
+        before { set_config "js_sort_translation_keys_false.yml"}
+
+        it { should eq true }
+      end
+    end
+
+    describe '.sort_translation_keys?' do
+      after { described_class.send(:remove_instance_variable, :@sort_translation_keys) }
```

Comment:
This needs to be performed after, similar to the describe '.export_i18n_js_dir_path' tests, in order to reset the instance variable so that it doesn't persist to the next test. This is why the incorrect false test pointed out above passed incorrectly.